ansible/roles_ocp_workloads/ocp4_workload_web_terminal/defaults/main.yml

```diff
@@ -1,5 +1,35 @@
+---
 become_override: false
 ocp_username: opentlc-mgr
 silent: false
-tmp_dir: /tmp/{{ guid }}
-tmp_kubeconfig: "{{ tmp_dir }}/.kube/config"
+
+# Channel to use for the Web Terminal subscription
+ocp4_workload_web_terminal_channel: fast
+
+
+# Set automatic InstallPlan approval. If set to false it is also suggested
+# to set the starting_csv to pin a specific version
+# This variable has no effect when using a catalog snapshot (always true)
+ocp4_workload_web_terminal_automatic_install_plan_approval: true
+
+# Set a starting ClusterServiceVersion.
+# Recommended to leave empty to get latest in the channel when not using
+# a catalog snapshot.
+# Highly recommended to be set when using a catalog snapshot but can be
+# empty to get the latest available in the channel at the time when
+# the catalog snapshot got created.
+ocp4_workload_web_terminal_starting_csv: "web-terminal.v1.8.0"
```

Comment:
Do you have to specify a starting CSV? I find that leaving this empty (as in "") is more future proof. OK to leave the version in a comment.

Reply:
Good point, I will change it.

Reply:
Thanks.